message-index/messages/GHC-88464/example1/index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Mistaken variable scope
+---
+
+In this example, the body of `g` attempts to refer to `x`, which is an argument to `f` and thus not available. The updated version renames `g`'s argument so that the `x` in the body can refer to it.
```

Comment:
I think most of the examples we have so far include a full error message, maybe that would be nice to have for this one as well?

Reply:
(I'd also be happy to add that in a followup PR of course.)

Reply:
Good idea - will do